rizwan171/trello-clone
# Updated dependencies and removed craco

3 main things with this PR:
- Updating react-scripts to 5.0.1
- Removing craco
- Updating some outdated libraries

Due to difficulties I faced with craco and jest, I've opted to return to using `react-scripts` instead of Craco. 
Craco was added way back when to allow us to use TailwindCss
However, with the newer react-scripts, you can run Tailwind without the need for Craco.


Note: you'll need to have yarn installed.
To install yarn, run `npm i -g yarn`
To test:
- run `yarn install`
- run `yarn start`
- confirm the app loads
